common/motion_utils/CMakeLists.txt

```diff
@@ -7,13 +7,14 @@ autoware_package()
 find_package(Boost REQUIRED)
 
 ament_auto_add_library(motion_utils SHARED
-  src/motion_utils.cpp
   src/distance/distance.cpp
   src/marker/marker_helper.cpp
   src/marker/virtual_wall_marker_creator.cpp
   src/resample/resample.cpp
+  src/trajectory/trajectory.cpp
   src/trajectory/interpolation.cpp
   src/trajectory/path_with_lane_id.cpp
+  src/trajectory/tmp_conversion.cpp
```

Comment:
Why is this temporary?

Reply:
Actually I do not know, `tmp_convesion.hpp` exists, so I followed the file name.

https://github.com/autowarefoundation/autoware.universe/blob/main/common/motion_utils/include/motion_utils/trajectory/tmp_conversion.hpp

Reply:
I see :+1: